dashboard/src/app/teams/team-details/team-details.controller.ts

```diff
@@ -169,50 +167,24 @@ export class TeamDetailsController {
       this.codenvyTeamEventsManager.removeDeleteHandler(deleteHandler);
     });
 
-    this.fetchTeamDetails();
-    this.fetchUser();
-  }
-
-  /**
-   * Fetches the team's details by it's name.
-   */
-  fetchTeamDetails(): void {
-    this.team  = this.codenvyTeam.getTeamByName(this.teamName);
-
-    if (!this.team) {
-      this.codenvyTeam.fetchTeamByName(this.teamName).then((team: any) => {
-        this.team = team;
-        this.newName = angular.copy(this.team.name);
+    this.team = teamDetailsService.getTeam();
```

Comment:
this part causes bug with typing another name in the address line:
![image](https://cloud.githubusercontent.com/assets/1611939/23654500/a7246d92-0339-11e7-853c-a99c0c02d1ce.png)